service/database.py

```diff
@@ -43,6 +43,14 @@ class ClassData(db.Model):
     pending = db.Column(db.Integer, nullable=False)
     total = db.Column(db.Integer, nullable=False)
 
+class ClassDetails(db.Model):
```

Comment:
Similar to App.py I think you could just add description as a new column for the ClassData class and remove the class ClassDetails to eliminate some redundancy, otherwise we'll have duplicate data for the other columns in this class since they're all present in ClassData, we'll also have to figure out a way to get the description of classes from NAU but that can prob be in a future release